cgi-bin/DW/Controller/API/REST/Entries.pm

```diff
@@ -0,0 +1,568 @@
+#!/usr/bin/perl
+#
+# DW::Controller::API::REST::Entries
+#
+# API controls for entries
+#
+# Authors:
+#      Allen Petersen <allen@suberic.net>
+#
+# Copyright (c) 2016 by Dreamwidth Studios, LLC.
```

Comment:
is this actually a 2016 file by Allen?

Reply:
Hilariously for once it actually is - he wrote the base code for the entry fetch and JSON transform of entry objects.